architecture/diagrams.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: "Novu Architecture Diagram"
+---
+# Novu Architecture Diagram powered by [terrastruct](https://terrastruct.com) and D2
+
+If the diagram below is not rendering, please [click here](https://app.terrastruct.com/diagrams/1895866270) to view it on terrastruct.com.
```

Comment:
So does this diagram capture version 0.21. or 0.22? @Cliftonz

Reply:
This document is a rolling document at the moment, currently it shows 0.21 but will be updated soon for 0.22

Reply:
Okay, so you will need to version this page. you can rename it to `diagrams-021.mdx`. 

and then add version like this:

```
---
title: "Novu Architecture Diagram"
version: 'v0.21.0'
---
```

@Cliftonz